### PR TITLE
Added integration for Hinge Angle sample with new screen manager SDK

### PR DESCRIPTION
**Pull request by soalb-m**
Added integration for Hinge Angle sample with new screen manager SDK

### CODE REVIEW

File: HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt [hunk 1]
```diff
@@ -7,45 +7,76 @@
...
+        if (screenInfo.isDualMode()) {
+            useDualMode(screenInfo)
+        } else {
+            useSingleMode()
+        }
+    }
 
-            supportFragmentManager
+    private fun useDualMode(screenInfo: ScreenInfo) {
+        Log.d(DEBUG_TAG, "########### dual mode")
```
Comment (CesarValiente) [inline HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt:55]: You could get rid of this debug logtrace

File: HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt [hunk 2]
```diff
@@ -7,45 +7,76 @@
...
+                        R.id.second_container_id,
+                        DualScreenFragment(),
+                        null
                 ).commit()
-        } else {
-            Log.d(DEBUG_TAG, "########### single mode")
-            supportFragmentManager
+    }
+
+    private fun useSingleMode() { // Setting layout for single portrait
```
Comment (CesarValiente) [inline HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt:72]: Just for single portrait? and single landscape? If it's for both cases probably good to remove the comment that can cause confusion.

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Sorin!

Just a couple of minor comments, please check them if they are worth to be addressed.

File: HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt [hunk 3]
```diff
@@ -7,45 +7,76 @@
```
Comment (CristianVerdes) [inline HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt:7]: No Microsoft copyright text.

**Review by CristianVerdes — APPROVED** [review]
Cesar already left you some comments.
Besides that and the copyrights, looks good to me.

File: HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/HingeAngleApplication.kt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package com.microsoft.device.display.samples.hingeangle
+
```
Comment (CesarValiente) [inline HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/HingeAngleApplication.kt:2]: Missed license header here

File: HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt (hunk 3 above)
Comment (CesarValiente) [reply HingeAngle/src/main/java/com/microsoft/device/display/samples/hingeangle/MainActivity.kt:7]: Still missed here

**Review by ancirja-m — APPROVED** [review]

**Pull request merged**